Formula/twilio.rb

```diff
@@ -3,8 +3,8 @@
 class Twilio < Formula
   desc "unleash the power of Twilio from your command prompt"
   homepage "https://github.com/twilio/twilio-cli"
-  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v3.0.0/twilio-v3.0.0.tar.gz"
   version "3.0.0"
```

Comment:
Will this support older versions? Since its still referring to 3.0.0 version. Also for each version the sha256 would be different. How will that be updated?